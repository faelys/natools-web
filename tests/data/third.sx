(elements
   (main-text "\
<p>Data in the third page.</p>\n"
(render page-comments))
   (title "Third Page"))
(tags (3 pages/news tags/topic1/other-subtopic tags/topic3))
