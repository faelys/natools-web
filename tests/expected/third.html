<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
  <head>
    <title>Third Page</title>
  </head>
  <body>
<h1>Third Page</h1>
<p>Data in the third page.</p>
<p>Validated on 2015-02-01T01:02:03+04:00</p>
    <div class="sidebar">
<h1>Side bar</h1>
<p>Tags: <a href="/tags/topic1">Secondary subtopic of topic I</a> <a href="/tags/topic3">Topic III</a></p>
<p>2nd group</p>
<p>Previous: <a href="/second">Second Page</a></p>
<p>Next: <a href="/fourth">Fourth Page</a></p>
    </div>
    <div class="footer">
<p>Default footer: <a href="/index">return to index</a></p>
    </div>
  </body>
</html>
